Serialiser_Engine/Compute/Serialise/DateTime.cs

```diff
@@ -0,0 +1,48 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2023, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Base;
+using MongoDB.Bson;
+using MongoDB.Bson.IO;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+
+namespace BH.Engine.Serialiser
+{
+    public static partial class Compute
+    {
+
+        /*******************************************/
+        /**** Public Methods                    ****/
+        /*******************************************/
+        private static void Serialise(this DateTime value, BsonDocumentWriter writer, Type targetType)
+        {
+            // TODO: is this right? will lose precision after round trip
```

Comment:
@adecler you put this comment here, you have any idea?

Reply:
Regarding this, this is behaving the same way as the previous serialiser was, see:

https://github.com/BHoM/BHoM_Engine/blob/a498583b58fa4513cdb541a1a6f0e27bc704d256/Serialiser_Engine/Objects/BsonSerializers/ObjectSerializer.cs#L86-L88

Hence think this can be marked as resolved, and if any issues is found with this, then a separate issue can be opened and resolved in a separate PR.